### PR TITLE
[devtools-translate] Spanish - What's New in DevTools (Chrome 100)

### PR DESCRIPTION
**Pull request by Caballerog**
[devtools-translate] Spanish - What's New in DevTools (Chrome 100)

### CODE REVIEW

**Comment by chybie** [comment]
Gracias Carlos! OVer to @midudev for review.

File: site/es/blog/new-in-devtools-100/index.md [hunk 1]
```diff
@@ -28,33 +27,50 @@ _Gracias [Miguel Ángel](https://midu.dev) por la traducción y [Carlos Caballer
 <!-- end: translation instructions -->
```
Comment (jecfish) [inline site/es/blog/new-in-devtools-100/index.md:27]: Please update the `whats-new.md` file too!

**Review by jecfish — CHANGES_REQUESTED** [review]

File: site/es/blog/new-in-devtools-100/index.md [hunk 2]
```diff
@@ -65,36 +81,59 @@ Chromium issues: [1222574](https://crbug.com/1222574), [1222573](https://crbug.c
...
+Puede personalizar el selector de una grabación si no utiliza los [selectores comunes](/docs/devtools/recorder/#selector).
+
+
+<!-- For example, this [demo page](https://jec.fyi/demo/recorder) uses the `data-automate` attribute as the selector. [Start a new recording](/docs/devtools/recorder/#record) and enter the `data-automate` as the selector attribute. Fill in an email address and observe the selector value (`[data-automate=email-address]`). -->
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/2PPPt9tOC2ZEz1l9F9AK.png", alt="Customize the recording’s selector", width="800", height="524" %}
+Por ejemplo, esta [página de demostración](https://jec.fyi/demo/recorder) usa el atributo `data-automate` como selector. [Iniciar una nueva grabación](/docs/devtools/recorder/#record) e ingrese `data-automate` como el atributo del selector. Complete una dirección de correo electrónico y observe el valor del selector (`[data-automate=email-address]`).
```
Comment (and-oli) [inline site/es/blog/new-in-devtools-100/index.md:109]: Same comment with regards to imperative form:
```suggestion
Por ejemplo, esta [página de demostración](https://jec.fyi/demo/recorder) usa el atributo `data-automate` como selector. [Inicie una nueva grabación](/docs/devtools/recorder/#record) e ingrese `data-automate` como el atributo del selector. Complete una dirección de correo electrónico y observe el valor del selector (`[data-automate=email-address]`).
```

**Review by and-oli — APPROVED** [review]
LGTM with minor observations. Gracias!

File: site/es/blog/new-in-devtools-100/index.md (hunk 1 above)
Comment (Caballerog) [reply site/es/blog/new-in-devtools-100/index.md:29]: Done!

**Review by jecfish — APPROVED** [review]

**Comment by jecfish** [comment]
Gracias all!

**Comment by Caballerog** [comment]
Go for version 5 (101)! :-P

**Pull request merged**